csharp/ql/test/library-tests/dataflow/global/DataFlow.expected

```diff
@@ -1,14 +1,11 @@
 | Capture.cs:12:19:12:24 | access to local variable sink27 |
 | Capture.cs:21:23:21:28 | access to local variable sink28 |
-| Capture.cs:30:19:30:24 | access to local variable sink29 |
```

Comment:
Hmm, something is wrong: we should see any changes to this file. Perhaps some stubs are missing?

Reply:
The stubs are there, but for some reason the model for "Select" seems be ignored by the test. If I manually implemeneted the body of Select the "missing" rows in the DataFlow.expected related to Select will re-appear.

Reply:
As discussed offline, this is because stubs are no longer treated as `fromLibrary`. https://github.com/github/codeql/pull/7314/commits/e9583fdf217a032ead1989748a249f63602a7d51 fixes this.